docs/technical-details/reference/graphql.md

```diff
@@ -0,0 +1,235 @@
+# GraphQL Querying
+
+HarperDB supports GraphQL in a variety of ways. It can be used for [defining schemas](../../developers/applications/defining-schemas.md), and for querying [Resources](./resource.md).
+
+Get started by setting `graphql: true` in `config.yaml`.
+
+This automatically enables a `/graphql` endpoint that can be used for GraphQL queries.
+
+> HarperDB's GraphQL component is inspired by the [GraphQL Over HTTP](https://graphql.github.io/graphql-over-http/draft/#) specification; however, it does not fully implement neither that specification nor the [GraphQL](https://spec.graphql.org/) specification.
+
+Queries can either be `GET` or `POST` requests, and both follow essentially the same request format. `GET` requests must use search parameters, and `POST` requests use the request body.
+
+For example, to request the GraphQL Query:
+```graphql
+query GetDogs {
+	Dog {
+		id
+		name
+	}
+}
+```
+The `GET` request would look like:
+```sh
+curl 'http://localhost:9926/graphql?query=query+GetDogs+%7B+Dog+%7B+id+name+%7D+%7D'
```

Comment:
I think it is a little easier to read HTTP requests in standard HTTP format, and less tech-specific (curl). I'd expect most modern front-end devs to actually be more familiar with `fetch` than curl. And plain HTTP is what we have used elsewhere in the REST docs.

Reply:
Scratch that, I like the look of http format better